devel/libical/Portfile

```diff
@@ -1,22 +1,19 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-cmake.out_of_source yes
-
 name                libical
 
 if {${subport} eq "${name}"} {
-	github.setup    libical libical 1.0.1 v
-	conflicts       libical-devel
+    github.setup    libical libical 2.0.0 v
+    conflicts       libical-devel
```

Comment:
Is this really a must? This affects only a handful of lines for this port... The required `git`-fu to insert an additional changeset just for this seems like overkill to me right now. :-)